package.json

```diff
@@ -207,12 +207,13 @@
     "yauzl": "^2.10.0"
   },
   "devDependencies": {
+    "@4lolo/resize-observer-polyfill": "^1.5.2",
```

Comment:
Why did we have to bring this in?

Reply:
Context: We talked about this on [the other PR](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1252#discussion_r815208754).

The, now unmaintained, original `resize-observer-polyfill` exports `ResizeObserver` without exporting `ResizeObserverCallback`. While that could be mitigated by using `@types/resize-observer-browser`, it causes [a different problem](https://github.com/que-etc/resize-observer-polyfill/issues/80) where its exported `contentRect` conflicts with TypeScript's own definition. `@4lolo/resize-observer-polyfill` [takes care](https://github.com/4lolo/resize-observer-polyfill/commit/af83516582034a5a45c24d097dc62a745c3c1e84) of the `contentRect` problem.